src/dev-spaces-preview/azext_dev_spaces_preview/__init__.py

```diff
@@ -19,19 +20,19 @@ def __init__(self, cli_ctx=None):
                                                         min_profile='2017-03-10-profile')
 
     def load_command_table(self, _):
-        with self.command_group('aks') as g:
-            g.custom_command('use-dev-spaces', 'aks_use_dev_spaces')
-            g.custom_command('remove-dev-spaces', 'aks_remove_dev_spaces')
+        with self.command_group('ads') as g:
```

Comment:
I thought the plan was to make these internal (so they don't appear as actual commands in az).

Reply:
That's the next update when we have other commands. Right not the extension should have some exposed commands to work with.